mappings/bnfdmd/tests/conftest.py

```diff
@@ -12,31 +13,43 @@
     / "BNF Snomed Mapping data 20241119.zip"
 )
 
+MOCK_FILENAME = "BNF Snomed Mapping data 20240101.zip"
+MOCK_QUOTED_FILENAME = quote(MOCK_FILENAME)
+MOCK_FILEPATH = f"2024-01/{MOCK_QUOTED_FILENAME}"
 
-def add_response(rsps, filename, release_date):
-    rsps.add(
-        responses.GET,
-        "https://www.nhsbsa.nhs.uk/prescription-data/understanding-our-data/bnf-snomed-mapping",
-        body=f"""
-            <p><a href="/sites/default/files/{release_date}/{filename.replace(' ','%20')}">January 2024 (ZIP file: 16.76MB)</a></p>
-            <p><a href="/sites/default/files/2023-01/BNF%20Snomed%20Mapping%20data%2020230116.zip">January 2023 (ZIP file: 16.76MB)</a></p>
-        """,
-        status=200,
-    )
+
+@pytest.fixture
+def mocked_responses_with_zipfile(mocked_responses):
+    with MOCK_BNFDMD_IMPORT_DATA_PATH.open("rb") as f:
+        mocked_responses.add(
+            responses.GET,
+            f"https://www.nhsbsa.nhs.uk/sites/default/files/{MOCK_FILEPATH}",
+            content_type="application/zip",
+            body=f,
+            status=200,
+        )
+        yield mocked_responses
 
 
 @pytest.fixture
 def mocked_responses():
     with responses.RequestsMock() as rsps:
+        rsps.add(
```

Comment:
I'm not sure I understand why this fixture needed to change in this commit. I think `test_get_latest_release_with_existing` is the only client and will pass without this change.

This is also a different meaning than `mocked_responses` in other `conftest.py` where it means something like "responses are just empty mocks". So if this specific fixture needs to return certain content can its name reflect that?